v2/azure-arm.yaml

```diff
@@ -2442,6 +2442,7 @@ objectModelConfiguration:
         UserAssignedIdentity:
           $armReference: false # Actually, this *IS* a resource id, but we want to avoid breaking changes so we're fibbing here
       KeyVaultProperties:
+        $nameInNextVersion: EncryptionKeyVaultProperties
         KeyVaultArmId:
           $armReference: false # Actually, this *IS* a resource id, but we want to avoid breaking changes so we're fibbing here
```

Comment:
Since we're already taking breaking changes here for `PrivateKeyData` and `PublicKeyData`, should we fix this up too? @matthchr, what do you think?

Reply:
I think we should avoid gratuitous breaking change unless there's a security reason behind it (like for the other properties).

For this one, ideally we fix this in a new API version - if we can do that here then yes that would be good to do.